docs/data/joy/components/tabs/tabs.md

```diff
@@ -161,30 +129,56 @@ You can use those to customize the component on both the `sx` prop and the theme
 
 ### Segmented controls
 
-To mimic the iOS segmented controls, add the border-radius to the `sx` prop of the TabList and set the selected Tab's background to `background.surface`.
+To mimic the iOS segmented controls, add a border-radius to the Tab List component and set the selected Tab background to `background.surface`.
 
 {{"demo": "TabsSegmentedControls.js"}}
 
 ### Browser tabs
 
-In this example, the Tab's variant is set to `outlined` and the indicator is moved to the top via `indicatorPlacement="top"`. The borders of the Tab are set to `transparent` based on the selected state.
+In this example, the Tab's variant prop is set to `outlined` and the indicator is moved to the top via `indicatorPlacement="top"`.
+The borders are then set to `transparent` based on the selected state.
 
 {{"demo": "TabsBrowserExample.js"}}
 
 ### Pricing tabs
 
-This example removes the background of the selected Tab by targeting `[aria-selected="true"]` on the `sx` prop.
+This example removes the background of the selected Tab by targeting the `aria-selected="true"` attribute.
 
 {{"demo": "TabsPricingExample.js"}}
 
-### With counter chips
+### Centralized tabs
```

Comment:
"Centralized" means something more like "consolidated" or "concentrated," when I think the word we're actually looking for here is "Centered"

```suggestion
### Centered tabs
```